### PR TITLE
Improves MSSQL testing

### PR DESCRIPTION
**Pull request by edgararuiz**

### CODE REVIEW

**Comment by edgararuiz** [comment]
PTAL @hadley  - Travis CI failed, but I think that's due to a previous master commit.

File: tests/testthat/test-translate-mssql.r [hunk 1]
```diff
@@ -1,183 +1,95 @@
...
+  expect_equal(trans(sd(x, na.rm = TRUE)),  sql("STDEV(`x`)"))
+  expect_equal(trans(var(x, na.rm = TRUE)), sql("VAR(`x`)"))
+
+  expect_error(trans(cor(x)), "not available")
+  expect_error(trans(cov(x)), "not available")
 })
 
-# stringr -------------------------------------------
+# MSSQL window conversions -----------------------------------------
+test_that("custom aggregators translated correctly", {
```
Comment (hadley) [inline tests/testthat/test-translate-mssql.r]: Should be custom _window_ functions in test name

File: tests/testthat/test-translate-mssql.r [hunk 2]
```diff
@@ -1,183 +1,95 @@
...
+    mf %>% filter(is.na(x)) %>% show_query(),
+    sql("SELECT *\nFROM `df`\nWHERE (((`x`) IS NULL))")
+  )
 
-test_that("Filter uses IS NULL for is.na ", {
-  expect_equivalent(
-    show_query(filter(df_mssql, is.na(x))),
-    sql("SELECT *\nFROM `df`\nWHERE (((`x`) IS NULL))"))
-})
+  expect_equal(
```
Comment (hadley) [inline tests/testthat/test-translate-mssql.r]: I don't think you need the tests of `!is.na()` here - it's just testing the composition of `!` and `is.na()`

File: tests/testthat/test-translate-mssql.r [hunk 3]
```diff
@@ -1,183 +1,95 @@
 context("translate-MSSQL")
 
-# MSSQL base_scalar conversions -----------------------------------------
+# MSSQL scalar conversions -----------------------------------------
```
Comment (hadley) [inline tests/testthat/test-translate-mssql.r]: You can now delete the section headers

**Comment by edgararuiz** [comment]
PTAL @hadley

File: tests/testthat/test-translate-mssql.r (hunk 2 above)
Comment (hadley) [reply tests/testthat/test-translate-mssql.r]: I think you missed this one?

Comment (edgararuiz) [reply tests/testthat/test-translate-mssql.r]: I just tested that query against an open MS SQL connection and it failed.  I need to run more "live" tests with a negated `is.na()`.  Is it ok for me to include the needed translations changes in this PR or should I do a separate one?

**Comment by edgararuiz** [comment]
To keep this PR clean, I'll submit a separate one to fix the `!is.na()` issue along with specific tests for it.

**Comment by hadley** [comment]
Perfect, thanks

**Pull request merged**